app/Http/Controllers/API/InviteController.php

```diff
@@ -14,16 +16,26 @@ public function __construct(protected InviteService $inviteService)
 
     }
 
-    public function sendInvite(InviteUserRequest $request)
+    public function sendInvite(InviteUserRequest $request): JsonResponse
     {
-        $validatedInviteUser = $request->validated();
+        $validatedInviteUser = $request->safe()->only(['name', 'email', 'role_id']);
 
         $inviteUserData = $this->inviteService->processInvite($validatedInviteUser);
 
         if(!$inviteUserData) {
```

Comment:
@sabinshrestha8 Remove this

Reply:
https://github.com/proshore/ProshoreAssetManagement-Backend/pull/12/commits/55d128b18cbdc90a0a866a427ab7a050af6791f5